README.md

```diff
@@ -44,64 +44,33 @@ benchmark [arguments (optional)]
-│ koa-router               │ 7.4.0   │ ✓      │ 31416      │ 3.1     │ 4.86          │
-├──────────────────────────┼─────────┼────────┼────────────┼─────────┼───────────────┤
-│ koa                      │ 2.5.1   │ ✗      │ 34750.4    │ 2.8     │ 5.47          │
-├──────────────────────────┼─────────┼────────┼────────────┼─────────┼───────────────┤
-│ micro-route              │ 2.5.0   │ ✓      │ 34622.4    │ 2.81    │ 5.44          │
-├──────────────────────────┼─────────┼────────┼────────────┼─────────┼───────────────┤
-│ micro                    │ 9.3.2   │ ✗      │ 35123.2    │ 2.77    │ 5.46          │
-├──────────────────────────┼─────────┼────────┼────────────┼─────────┼───────────────┤
-│ microrouter              │ 3.1.3   │ ✓      │ 25940.8    │ 3.76    │ 4.14          │
-├──────────────────────────┼─────────┼────────┼────────────┼─────────┼───────────────┤
-│ polka                    │ 0.4.0   │ ✓      │ 43158.4    │ 2.25    │ 6.08          │
-├──────────────────────────┼─────────┼────────┼────────────┼─────────┼───────────────┤
-│ rayo                     │ 1.0.4   │ ✓      │ 44668.8    │ 2.17    │ 4.97          │
-├──────────────────────────┼─────────┼────────┼────────────┼─────────┼───────────────┤
-│ restify                  │ 7.2.1   │ ✓      │ 26945.6    │ 3.47    │ 4.28          │
-├──────────────────────────┼─────────┼────────┼────────────┼─────────┼───────────────┤
-│ server-base-router       │ 6.0.0   │ ✓      │ 44252.8    │ 2.19    │ 4.97          │
-├──────────────────────────┼─────────┼────────┼────────────┼─────────┼───────────────┤
-│ server-base              │ 6.0.0   │ ✗      │ 43177.6    │ 2.25    │ 4.83          │
-├──────────────────────────┼─────────┼────────┼────────────┼─────────┼───────────────┤
-│ spirit-router            │ 0.5.0   │ ✓      │ 32124.8    │ 2.9     │ 5.03          │
-├──────────────────────────┼─────────┼────────┼────────────┼─────────┼───────────────┤
-│ spirit                   │ 0.6.1   │ ✗      │ 36726.4    │ 2.51    │ 5.72          │
-├──────────────────────────┼─────────┼────────┼────────────┼─────────┼───────────────┤
-│ take-five                │ 1.4.0   │ ✓      │ 23025.6    │ 4       │ 7.63          │
-├──────────────────────────┼─────────┼────────┼────────────┼─────────┼───────────────┤
-│ total.js                 │ 2.9.4   │ ✓      │ 26945.6    │ 3.62    │ 7.60          │
-├──────────────────────────┼─────────┼────────┼────────────┼─────────┼───────────────┤
-│ trek-engine              │ 1.0.5   │ ✗      │ 37462.4    │ 2.6     │ 5.28          │
-├──────────────────────────┼─────────┼────────┼────────────┼─────────┼───────────────┤
-│ trek-router              │ 1.2.0   │ ✓      │ 37339.2    │ 2.61    │ 5.33          │
-├──────────────────────────┼─────────┼────────┼────────────┼─────────┼───────────────┤
-│ vapr                     │ 0.5.1   │ ✓      │ 35464      │ 2.74    │ 5.03          │
-└──────────────────────────┴─────────┴────────┴────────────┴─────────┴───────────────┘
-```
```

Comment:
This seems to remove the opening backticks

Reply:
This is intentionally. This moves the table to the correct format for markdown to work so that GitHub will automatically format the table.